test/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
+$VERBOSE = true
+
+if Warning.respond_to?(:[]=)
+  Warning[:deprecated] = true
```

Comment:
I'm curious, does setting this to true _disable_ all deprecation warnings? If so, would we see these warnings when running `gem build` (I forget)? We should probably see them _somewhere_.

Reply:
> does setting this to true _disable_ all deprecation warnings?

No, it turns them on actually. Starting from ruby `2.7.2` deprecation warnings are off by default. So it's good practice to turn them on in your test suite. Ultimately the intent is to have test frameworks such as minitest to do it by default, but until then...